### ISSUE TITLE
[Feature request] Ability to disable integrated photo viewer

### ISSUE THREAD

**Issue by matejdro** [telegramdesktop/tdesktop#1111]
When you receive images from chat (either sent as image or as file), they are displayed in integrated photo viewer.

This viewer lacks zoom and other functionality that is present in other photo viewers. So if I want this functionality I must right click on the image, select "View in folder" and then open file manually from file browser.

There should be an option to disable integrated viewer and open my default image viewer when I click on the image in chat.

**Comment by auchri** [comment]
Zoom with ctrl and +/-

**Comment by matejdro** [comment]
Oh wow I didn't even tried it. Why wouldn't you support mouse scroll wheel zoom?

**Comment by joshery420** [comment]
Agreed. Ctrl +/- works, but mouse-wheel zoom is much more intuitive, especially when people usually open images in the viewer with a mouse click, not keyboard input. Since we use the mouse to open and close images, should use the mouse for most image interaction.

**Comment by Aokromes** [comment]
you can do mouse wheel zoom now.

**Comment by Aokromes** [comment]
now we have https://github.com/telegramdesktop/tdesktop/commit/54c409d967f9e823afdb8d9f94d775074c284aa0 maybe it can be improved to include photo viewers / audio players, etc without being hidden setting.

**Comment by kirsan31** [comment]
> now we have [54c409d](https://github.com/telegramdesktop/tdesktop/commit/54c409d967f9e823afdb8d9f94d775074c284aa0) maybe it can be improved to include photo viewers / audio players, etc without being hidden setting.

This _simple_ PR works only with **downloaded** (on disk as video file) video. In theory same must go with photos, **but**  in practice in TD there are no **downloaded** (in same therms as video) photo :( This button:
![image](https://user-images.githubusercontent.com/17767561/73193825-3918ef00-413c-11ea-9c9b-d571bdffaf18.png)

always download photo and save it with **new name**. So, with photo it's of course doable, but not so simple as with video...

**Comment by matejdro** [comment]
We are still interested in this.

**Comment by kirsan31** [comment]
Still actual.

**Comment by qsmodo** [comment]
Zoom works but there is no way to scroll the image (e.g. to see a detail in a corner) as far as I can tell. Saving the picture and opening it in an external viewer calls for a bunch of clicks and context-switching.

**Comment by ilya-fedin** [comment]
What @john-preston meant (I guess) is that it's implemented in a way that it's not saved to disk and it's hard to change that (read like "too much effort for such a low priority issue"), not that it's some security feature that require some question to decrypt images, it's not a secret chat or something, you can still save the file by a button in the viewer. Also, if a feature requires an option, it's a blocker in tdesktop. Adding options is not something @john-preston likes.

**Comment by aljustiet** [comment]
The same request

**Comment by Aokromes** [comment]
comming on next version.
